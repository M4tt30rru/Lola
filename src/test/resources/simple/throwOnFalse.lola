##Import "std.lola"
##Find ##Expression(_cond) ?! ##Identifier(_exc);
 ##replace if (!(##(_cond))) throw new ##(_exc)(); else ;

f() ?! BadTimeException;
