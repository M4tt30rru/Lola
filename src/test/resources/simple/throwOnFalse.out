if (!(f()))
 throw new BadTimeException();
else
 ;